lib/provider.js

```diff
@@ -73,5 +94,8 @@ module.exports = {
   },
   featureDirError: function(rootDirectory=this.rootDirectory(), featuresDir=this.featuresDirectory()) {
     throw new Error(`Cannot find features directory at ${rootDirectory}${featuresDir}`);
+  },
+  searchForPattern: function(path, pattern){
+    return search(path, pattern, 0);
```

Comment:
Looks like third argument `0` is not required, as it is the default in `search` method signature :)